app/src/main/java/edu/uc/app/parkfinder/dto/Park.kt

```diff
@@ -2,8 +2,6 @@ package edu.uc.app.parkfinder.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Park(@SerializedName("Code") var code: String,
-                @SerializedName("Name") var name: String) {
-
+data class Park(@SerializedName("Code") var code: String, @SerializedName("Name") var name: String) {
```

Comment:
Unless variable declarations take more than a screen of space, I would keep it to one line. For 2 variables, it would be acceptable and readable to leave in one line.

Reply:
Formatting is a user preference, and can be set by the IDE automatically... so I tend not to change a user's preference.  It's safe to leave this as it were.